tool/github-action/sass-spec-ref.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+# Copyright 2016 Google Inc. Use of this source code is governed by an MIT-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+# Echoes the sass-spec Git ref that should be checked out for the current Travis
+# run. If we're running specs for a pull request which refers to a sass-spec
+# pull request, we'll run against the latter rather than sass-spec master.
+
+GITHUB_REF=${PR_REF:-$CURRENT_REF}
+if [[ "$GITHUB_REF" == refs/heads/feature.* ]]; then
+  default="${GITHUB_REF:11}"
+else
+  default=master
+fi
+
+# We don't have a PR_REF so we are not in a pull request
+if [ "$GITHUB_REF" == "$CURRENT_REF" ]; then
+  >&2 echo "Ref: $default."
+  echo "$default"
+  exit 0
+fi
+
+>&2 echo "$PR_BODY"
+
+RE_SPEC_PR="sass/sass-spec(#|/pull/)([0-9]+)"
+
+if [[ $PR_BODY =~ $RE_SPEC_PR ]]; then
```

Comment:
Very exciting to be able to do this without an HTTP request :smiley:.

should `$PR_BODY` and `$RE_SPEC_PR` be quoted here?

Reply:
there was no quotes around them in the previous script, which I took as basis for this one.

Reply:
Doing some poking around in Bash documentation, I think `$PR_BODY` should be quoted but `$RE_SPEC_PR` should not:

```suggestion
if [[ "$PR_BODY" =~ $RE_SPEC_PR ]]; then
```